### ISSUE TITLE
[BUG]: Unable to scan local html files with a11y checker 

### ISSUE THREAD

**Issue by eirckr** [IBMa/equal-access#871]
### Project

a11y checker extension

### Browser

Chrome

### Operating system

MacOS

### Description

User is not able to scan local html files with checker

### Steps to reproduce

1. Open a local html file in chrome browser
2. open the a11y checker and try to scan 

<img width="1576" alt="Screen Shot 2022-05-06 at 12 07 10 PM" src="https://user-images.githubusercontent.com/62436670/167179567-9a1a5e27-6dda-41be-b62e-3be6e7e5740c.png">

**Comment by tombrunet** [comment]
Did you enable access to file URLs in Chrome? This is a Chrome extension setting:
<img width="716" alt="image" src="https://user-images.githubusercontent.com/19577713/167182418-72768806-f5cb-4ae3-b86f-9c5c83a89a90.png">

**Comment by eirckr** [comment]
Yes, we have access to local file enable

**Comment by doveye** [comment]
I'm using Chrome on Windows 10. On 10 May, I successfully checked a couple of local files, which I left as open tabs in Chrome. I put my laptop into sleep mode and on 11 May, the checker would not check the files again, giving the same message as seen in the screenshot above. I've tried refreshing the browser tabs, re-opening the browser developer tools, closing the tabs then restarting the browser then re-opening the files, re-selecting the the "Allow access to file URLs" extension option - nothing worked. I haven't tried rebooting the machine because I don't want to shut it down right now. It's odd that it worked then the next day it didn't.

**Comment by shunguoy** [comment]
reproducible, high priority

**Comment by drjoho** [comment]
FYI was unable to work on this Thur as Build did not work. Build started working again 10pm Thur. Will work on it Tue.

**Comment by drjoho** [comment]
I added "<all_urls>" to host_permissions in the manifest.json file. Which allows access to local files if as stated above This is a Chrome extension setting:
Allow access to file urls is set.

**Comment by eirckr** [comment]
Unable to scan local file. 
![Uploading Screen Shot 2022-07-05 at 1.28.20 PM.png…]()

**Comment by drjoho** [comment]
Fixed. Changes needed to be in manifest_Chrome.json. Done. @eirckr please test again.
